cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -14,14 +14,39 @@
 
 from typing import (Any, Dict, Iterable, List, NamedTuple, Optional, Sequence,
                     Set, Tuple, Union)
+import dataclasses
 import numpy as np
+import scipy
 from matplotlib import pyplot as plt
 from cirq import circuits, devices, ops, protocols, sim, work
 
 CrossEntropyPair = NamedTuple('CrossEntropyPair', [('num_cycle', int),
                                                    ('xeb_fidelity', float)])
 
 
+@dataclasses.dataclass
+class DepolarizingModel:
+    """A model of depolarizing noise.
+
+    Models the fidelity of a quantum process as an exponential decay of the form
+    f = S * p^d, where f is the fidelity, S is the coefficient, p is the
```

Comment:
Is "f" the average or entanglement fidelity?

Reply:
I think it's neither. I rewrote the documentation to better describe S and p, and I'm no longer using the term "fidelity".